### ISSUE TITLE
TargetInvocationException on vote for Spouse (indebted Lord start)

### ISSUE THREAD

**Issue by kinith** [R-Vaccari/bannerlord-banner-kings#116]
---
name: Bug report
about: Create a report to help us improve
title: 'TargetInvocationException on vote for Spouse (indebted Lord start)'
labels: bug
assignees: ''

---

**Describe the bug**
Long form:
After starting as indebted lord on version 1.2.7.2-1 I get bombarded at the start with votes for the different council positions, as far as I could tell the others work fine, however when the vote starts for the spouse position (which btw doesn't have any entry except abstain) the game crashes with the exception listed in the report. 

I would assume that since it is a divide by zero exception that the calculation for the percentage on how many lords voted for which part had a problem with the fact that there was only one option. 

**TLDR**
Steps for the crash to happen
- Start as indebted lord (northern Empire, but I guess it will happen for everyone)
- Save and load to enable voting functionality
- Wait for Council positions to be filled
- Crash on Spouse position voting. (Which has no entry except abstain)
- Divide by Zero Exception

Other minor things I noticed while testing:
- Voting seems to only be possible after saving and reloading
- Spouse voting seems to only happen sometimes, as somtimes the spouse simply gets selected without a vote
- Votes only seem to be triggered outside of a town, so no save waiting inside a town to wait for this crash

**Mod Version**
- Standard I assume... Not quite sure what you mean with CEK version?
- Numeric Version: 1.2.7.2-1

**Evidence**
- [x] The mod is present in the callstack;
- [x] The issue only happens with the mod activated;
- [x] You tested it with the least amount of other mods as possible.

**Crash Report**
https://report.butr.link/49708F.html

Edit: Reformated towards the template, put in the informations from my comment below and added a TLDR portion that shortly goes through the steps that lead to the crash.

**Comment by kinith** [comment]
Finally.... after 2 hours of testing.... The event triggered again after I disabled most of the mods....

What I noticed: 
- As stated above, voting seems to only be possible after saving and reloading
- Spouse voting seems to only happen sometimes, as somtimes the spouse simply gets selected without a vote
- Votes only seem to be triggered outside of a town, so no save waiting inside a town to wait for this crash 

Full report
https://report.butr.link/49708F.html

**Comment by R-Vaccari** [comment]
I added a check for candidates count. Should not happen anymore as the election takes at least 3 candidates as Bannerlord expects. This is for next update after this date.
